allennlp/data/fields/sequence_field.py

```diff
@@ -1,7 +1,8 @@
-from allennlp.data.fields.field import Field
+from typing import Generic
+from allennlp.data.fields.field import Field, DataArray
 
 
-class SequenceField(Field):
+class SequenceField(Field[DataArray], Generic[DataArray]):
```

Comment:
actually, I take that back, that issue is closed, it should work with just the first one

https://github.com/python/mypy/pull/2811

@DeNeutoy sorry for directing you wrong